translations/de.ts

```diff
@@ -598,69 +579,42 @@ Soll der Proxyserver ignoriert werden und eine Direktverbindung hergestellt werd
 <context>
     <name>Core</name>
     <message>
-        <location filename="../src/core/core.cpp" line="260"/>
+        <location filename="../src/core/core.cpp" line="259"/>
         <source>Toxing on qTox</source>
-        <translation>Tox ist Toll!</translation>
+        <translation>Tox ist toll</translation>
```

Comment:
I'm with @antis81. Actually, it's not that important. :wink:
But "Toxt mit qTox" sound a little bit strange for me.